master/internal/scheduler/kubernetes_resource_provider.go

```diff
@@ -227,14 +227,20 @@ func (k *kubernetesResourceProvider) scheduleTask(ctx *actor.Context, task *Task
 			return
 		}
 
-		if task.SlotsNeeded()%k.config.SlotsPerNode != 0 {
-			ctx.Log().WithField("task-id", task.ID).Errorf(
-				"task number of slots (%d) is not schedulable on the configured "+
-					"slots_per_node (%d)", task.SlotsNeeded(), k.config.SlotsPerNode)
-			return
+		if task.SlotsNeeded() <= k.config.SlotsPerNode {
```

Comment:
So `k.config.SlotsPerNode` is always constant for k8s? Good to know.

Reply:
Yep its defined when the helm chart is installed / upgraded.